global/matlab/mixtureModel.m

```diff
@@ -27,11 +27,11 @@ function mixtureModel(inFile,outFile,wbcmd,melocmd)
 % Save melodic log?
 
 %% handle inputs 
-if nargin < 2 || isempty(outFile); outFile = inFile; end 
+if nargin < 2 || isempty(inFile) || isempty(outFile); help mixtureModel; return; end 
 if nargin < 3 || isempty(wbcmd); wbcmd = 'wb_command'; end 
 if nargin < 4 || isempty(melocmd); melocmd = 'melodic'; end 
-[wbStat,~] = unix(wbcmd);
-[meloStat,~] = unix(['which ' melocmd]);
+[wbStat,~] = system(wbcmd);
```

Comment:
Using `which` is probably a better idea than trusting that wb_command will never treat "no arguments, so print the help info" as a non-error condition.

Reply:
I suppose `-version` would be fairly safe, and would also work on windows.  But, as long as the melodic check uses `which`, might as well use the same approach for both.